Document issue registry issue persistence (#1459)
home-assistant/developers.home-assistant: docs/issue_registry_index.md

)
```

## Issue life cycle

### Issue persistence

> An issue will be kept in the issue registry until it's removed by the integration creating it or by the user [fixing](#fixing-an-issue) it.

Reviewer: ```suggestion
An issue will be kept in the issue registry until it's removed by the integration that created it or by the user [fixing](#fixing-an-issue) it.
```

The `is_persistent` flag controls if an issue should be shown to the user after a restart of Home Assistant:

> - If the `is_persistent` flag is set on the issue, the issue will be shown again to the user after a restart.

Reviewer: ```suggestion
- If the `is_persistent` flag is set on the issue, the issue will be shown again to the user after a restart. Use this for issues that can only be detected when they occur (update failed, unknown service in automation).
```

> - If the `is_persistent` flag is not set on the issue, the issue will not be shown again to the user after a restart until it's created again by its integration.

Reviewer: ```suggestion
- If the `is_persistent` flag is not set on the issue, the issue will not be shown again to the user after a restart until it's created again by its integration. Use this for issues that can be checked for, like low disk space.
```
